go/bind/notifications.go

```diff
@@ -80,8 +80,8 @@ func HandlePostTextReply(strConvID, tlfName string, intMessageID int, body strin
 
 func HandleBackgroundNotification(strConvID, body, serverMessageBody, sender string, intMembersType int,
 	displayPlaintext bool, intMessageID int, pushID string, badgeCount, unixTime int, soundName string,
-	pusher PushNotifier) (err error) {
-	if err := waitForInit(5 * time.Second); err != nil {
+	pusher PushNotifier, dontShowIfStale bool) (err error) {
```

Comment:
`showIfStale`, please, so we don't have to interpret the double negative `dontShowIfStale == false` below.